plv8_func.cc

```diff
@@ -186,15 +178,6 @@ SubTranBlock::exit(bool success)
 		ReleaseCurrentSubTransaction();
 	else
 		RollbackAndReleaseCurrentSubTransaction();
-
-	MemoryContextSwitchTo(m_mcontext);
-	CurrentResourceOwner = m_resowner;
-
-	/*
-	 * AtEOSubXact_SPI() should not have popped any SPI context, but just
-	 * in case it did, make sure we remain connected.
-	 */
-	SPI_restore_connection();
```

Comment:
ok, i'll do some quick refresh reading - in the mean time, i went ahead and fixed the OS X builds in a separate pull request, and have been working to try to get travis passing again (about 12 hours down into that rabbit hole right now, there were a number of issues)